devops/azure-pipelines/build-stage-template.yml

```diff
@@ -0,0 +1,17 @@
+stages:
+- stage: Build
+  displayName: ${{ parameters.displayName }}
+  jobs:
+  - job: Build
+    steps:
+    - task: NodeTool@0
+      inputs:
+        versionSpec: '10.x'
+      displayName: 'Install Node.js'
+
+    - script: |
```

Comment:
don't run install and build in the same step like this, because by default, if npm install fails, the error will be ignored just as if this yaml string value was run as a bash script (because it is). yes, if install fails, build will also happen to fail as a consequence and _that_ exit code will be seen by the job, but it's semantically bad. i realize these issues are coming from the template yaml, but they should still be addressed.